lib/active_fedora/indexing.rb

```diff
@@ -82,13 +82,33 @@ def index_config
                             end
         end
 
-        def reindex_everything
+        def reindex_everything(batch_size: 50, softCommit: true, progress_bar: false, final_commit: false)
```

Comment:
No problem, will do. Thanks for giving me the boilerplate, I wasn't sure what format for arg documentation was being used or where it was documented. 

Btw, github has an "allow maintainers to make commits to your fork branch" option on PR's now, which I always check. So if it would have been easier to just go and make this commit in my branch instead of directing me to do it in a code-by-wire sort of thing, you are welcome to in the future!